# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnofract4d/install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur gnofract4d-3.1_orig/fract4d/test_fractal.py gnofract4d-3.1/fract4d/test_fractal.py
--- gnofract4d-3.1_orig/fract4d/test_fractal.py	2006-09-09 01:51:37.000000000 +0100
+++ gnofract4d-3.1/fract4d/test_fractal.py	2006-09-09 01:51:43.000000000 +0100
@@ -1083,7 +1083,7 @@
 @bailfunc=cmag
 @_gradient=[
 GIMP Gradient
-Name: /usr/share/gimp/1.2/gradients/Abstract_3
+Name: /opt/gnome2/share/gimp/2.0/gradients/Abstract_3
 6
 0.000000 0.050083 0.435726 0.000000 0.424242 0.070751 1.000000 1.000000 0.725647 0.428066 1.000000 0 0
 0.435726 0.490818 0.590985 1.000000 0.725647 0.428066 1.000000 0.115248 0.249315 0.651515 1.000000 0 0
diff -ur gnofract4d-3.1_orig/fract4dgui/colors.py gnofract4d-3.1/fract4dgui/colors.py
--- gnofract4d-3.1_orig/fract4dgui/colors.py	2006-09-09 01:51:37.000000000 +0100
+++ gnofract4d-3.1/fract4dgui/colors.py	2006-09-09 01:51:43.000000000 +0100
@@ -55,7 +55,7 @@
             os.path.join(sys.exec_prefix,"share/maps/gnofract4d"))
 
         # find gimp gradient files
-        gimp_dir = os.path.join(sys.exec_prefix,"share/gimp/")
+        gimp_dir = os.path.abspath("/opt/gnome2/share/gimp/2.0")
         if os.path.isdir(gimp_dir):
             for gimp_ver in os.listdir(gimp_dir):
                 self.add_directory(
diff -ur gnofract4d-3.1_orig/fract4dgui/main_window.py gnofract4d-3.1/fract4dgui/main_window.py
--- gnofract4d-3.1_orig/fract4dgui/main_window.py	2006-09-09 01:51:37.000000000 +0100
+++ gnofract4d-3.1/fract4dgui/main_window.py	2006-09-09 01:53:12.000000000 +0100
@@ -1231,8 +1231,8 @@
         loc = "C" # FIXME
 
         # look locally first to support run-before-install
-        local_dir = "doc/gnofract4d-manual/%s/" % loc
-        install_dir = "share/gnome/help/gnofract4d/%s/" % loc
+        local_dir = "share/doc/gnofract4d/gnofract4d-manual/%s/" % loc
+        install_dir = "share/doc/gnofract4d/gnofract4d-manual/%s" % loc
 
         helpfile = utils.find_resource(base_help_file, local_dir, install_dir)
         abs_file = os.path.abspath(helpfile)
diff -ur gnofract4d-3.1_orig/setup.py gnofract4d-3.1/setup.py
--- gnofract4d-3.1_orig/setup.py	2006-09-09 01:51:37.000000000 +0100
+++ gnofract4d-3.1/setup.py	2006-09-09 01:56:36.000000000 +0100
@@ -193,12 +193,6 @@
            ('share/formulas/gnofract4d',
             get_files("formulas","frm") + get_files("formulas", "ucl")),
 
-           # documentation
-           ('share/gnome/help/gnofract4d/C',
-            get_files("doc/gnofract4d-manual/C", "xml")),
-           ('share/gnome/help/gnofract4d/C/figures',
-            get_files("doc/gnofract4d-manual/C/figures",".png")),
-
            #internal pixmaps
            ('share/pixmaps/gnofract4d',
             ['pixmaps/deepen_now.png',
@@ -213,10 +207,6 @@
 
            # MIME type registration
            ('share/mime/packages', ['gnofract4d-mime.xml']),
-           
-           # doc files
-           ('share/doc/gnofract4d-%s/' % gnofract4d_version,
-            ['COPYING', 'README']),
            ],
        cmdclass={
            "my_bdist_rpm": my_bdist_rpm.my_bdist_rpm,
